app/services/delivery_partners/participants_filter.rb

```diff
@@ -36,27 +37,25 @@ def filter_query(scoped, query)
         user_full_name
         user_email
         user_teacher_profile_trn
-        induction_records_induction_programme_partnership_lead_provider_name
-        induction_records_school_cohort_school_name
-        induction_records_school_cohort_school_urn
+        induction_programme_partnership_lead_provider_name
+        school_cohort_school_name
+        school_cohort_school_urn
       ].join("_or_")
 
       scoped.includes(
```

Comment:
suggestion: whilst filtering by mentor/academic year there are a lot of n+1 coming up (some queries however are cahed and some aren't) not sure if we're missing an includes here?

Reply:
Good catch, I've fixed up a commit with additional includes. It might be worth considering a gem like goldiloader for our app, it would simplify the queries we have around filtering by handling the includes for us.